lib/blocs/bloc_provider.dart

```diff
@@ -18,12 +18,10 @@ class BlocProvider<T extends BlocBase> extends StatefulWidget {
   _BlocProviderState<T> createState() => _BlocProviderState<T>();
 
   static T of<T extends BlocBase>(BuildContext context) {
-    final type = _typeOf<BlocProvider<T>>();
-    BlocProvider<T> provider = context.ancestorWidgetOfExactType(type);
+    BlocProvider<T> provider =
+        context.findAncestorWidgetOfExactType<BlocProvider<T>>();
```

Comment:
does this change not break something?

Reply:
For me, it did not change anything. Unfortunatly the PR is again broken because I try to sync Lokalise with the PR. It's quite a mess. But after that, it would be great if you could test it.

Reply:
🎉Should work again.